webpack.config.js

```diff
@@ -1,61 +1,8 @@
-const path = require('path');
-
-const katexConfig = {
-    entry: ['./katex.js'],
-    output: {
-        path: path.join(__dirname, 'build'),
-        filename: 'katex.js',
-        library: 'katex',
-        libraryTarget: 'umd',
-    },
-};
-
-const copyTexConfig = {
-    entry: ['./contrib/copy-tex/copy-tex.js'],
-    output: {
-        path: path.join(__dirname, 'build', 'contrib', 'copy-tex'),
-        filename: 'copy-tex.js',
-    },
-};
-
-const autoRenderConfig = {
-    entry: ['./contrib/auto-render/auto-render.js'],
-    output: {
-        path: path.join(__dirname, 'build', 'contrib', 'auto-render'),
-        filename: 'auto-render.js',
-        library: 'renderMathInElement',
-        libraryTarget: 'umd',
-    },
-};
-
-const commonConfig = {
-    module: {
-        rules: [
-            {
-                test: /\.js$/,
-                use: 'babel-loader',
-                exclude: /\bnode_modules\b/,
-            },
-        ],
-    },
-    devtool: 'eval-source-map',
-};
-
-module.exports = {
-    compilerConfig: [
-        Object.assign({}, katexConfig, commonConfig),
-        Object.assign({}, copyTexConfig, commonConfig),
-        Object.assign({}, autoRenderConfig, commonConfig),
-    ],
-    devServerConfig: {
-        publicPath: '/',
-        contentBase: path.join(__dirname, 'build'),
-        stats: {
-            colors: true,
-        },
-        // Allow server to be accessed from anywhere, which is useful for
-        // testing.  This potentially reveals the source code to the world,
-        // but this should not be a concern for testing open-source software.
-        disableHostCheck: true,
-    },
-};
+// @flow
+const { targets, createConfig } = require('./webpack.common');
+
+module.exports = targets.reduce((configs, target) => {
+    configs.push(createConfig(target, false, false));
+    configs.push(createConfig(target, false, true));
+    return configs;
+}, []);
```

Comment:
I have a preference for avoiding `reduce`.  Maybe we could do:
```
module.exports = [
    ...targets.map(target => createConfig(target, false, false)),
    ...targets.map(target => createConfig(target, false, true)),
];
```
It isn't necessary to to interleave the configs is it?

Reply:
It might also be good to have comments here to remind people what the booleans are for.

Reply:
@kevinbarabash No, the order doesn't matter as it runs them in parallel. Will change and add comments.